src/migrations/scripts/1597921613888-initial.ts

```diff
@@ -604,7 +604,19 @@ const migrationCommands = function (transaction: any): Commands {
           creationDate: {
             type: Sequelize.DATE,
             field: 'creationDate'
-          }
+          },
```

Comment:
This should be new migration

Reply:
This has been addressed in the description. In fact, this is the reason it’s still a draft.